sdk/identity/azure-identity/src/samples/Azure-Cache-For-Redis/Jedis/Azure-AAD-Authentication-With-Jedis.md

```diff
@@ -45,27 +52,27 @@ When migrating your existing your application code, you need to replace the pass
 Integrate the logic in your application code to fetch an Azure AD access token via the Azure Identity library as shown below and replace it with the password configuring/retrieving logic in your application code.
 
 ```java
-//Construct a Token Credential from Identity library, e.g. ClientSecretCredential / Client CertificateCredential / ManagedIdentityCredential etc.
-ClientCertificateCredential clientCertificateCredential = new ClientCertificateCredentialBuilder()
-        .clientId("YOUR-CLIENT-ID")
-        .pfxCertificate("YOUR-CERTIFICATE-PATH", "CERTIFICATE-PASSWORD")
-        .tenantId("YOUR-TENANT-ID")
-        .build();
+//Construct a Token Credential from Identity library, e.g. DefaultAzureCredential / ClientSecretCredential / Client CertificateCredential / ManagedIdentityCredential etc.
+DefaultAzureCredential defaultAzureCredential = new DefaultAzureCredentialBuilder().build();
 
 // Fetch an Azure AD token to be used for authentication. This token will be used as the password.
-String token = clientCertificateCredential
-        .getToken(new TokenRequestContext()
-                .addScopes("https://*.cacheinfra.windows.net:10225/appid/.default")).block().getToken();
+// Note: The Scopes parameter will change as the Azure AD Authentication support hits public preview and eventually GA's.
+String token = defaultAzureCredential
+    .getToken(new TokenRequestContext()
+        .addScopes("https://*.cacheinfra.windows.net:10225/appid/.default")).block().getToken();
 
-// SSL connection is required for non 6379 ports.
-boolean useSsl = true; 
-String cacheHostname = "YOUR_HOST_NAME.redis.cache.windows.net";
+// SSL connection is required.
+boolean useSsl = true;
+// TODO: Replace Host Name with Azure Cache for Redis Host Name.
+String cacheHostname = "<HOST_NAME>";
 
 // Create Jedis client and connect to the Azure Cache for Redis over the TLS/SSL port using the access token as password.
+// Note, Redis Cache Host Name and Port are required below
```

Comment:
If they're required why doesn't the API enforce that? If I was trying to use the API a comment like this would make me wonder what else I need to double check with documentation versus the API making me do the right thing.

Reply:
The requirement is for Azure AD scenarios, 
Since we're only building samples and not custom API in this case, we 're bounded by OSS library design.
So, as per discussion with Redis team, we decided to call out required parameters in comment as guidance for users.

Reply:
Makes sense.